lms/djangoapps/discussion/static/discussion/js/discussion_router.js

```diff
@@ -18,7 +18,7 @@
                 initialize: function(options) {
                     Backbone.Router.prototype.initialize.call(this);
                     _.bindAll(this, 'allThreads', 'showThread');
-                    this.courseId = options.courseId;
+                    this.rootUrl = options.rootUrl;
```

Comment:
1. As I wrote above, is `options` different here?
2. I asked because I had seen `this.course_settings`. but I guess that really should be `this.courseSettings`?